app/views/layouts/application.html.erb

```diff
@@ -16,12 +16,13 @@
             <% if current_user %>
               <li ><%= link_to "Sign out", destroy_user_session_path, method: :delete %></li>
               <li ><%= link_to "Account", edit_user_registration_path %></li>
+              <li ><%= link_to "Topics", topics_path %></li>
             <% else %>
-              <li role="presentation"><%= link_to "Sign in", new_user_session_path %></li>
+              <li><%= link_to "Sign in", new_user_session_path %></li>
+              <li><%= link_to "Sign up", new_user_registration_path %></li>
             <% end %>
         </ul>
       </nav>
-      </div>
 
       <% if flash[:notice] %>
         <div class="alert alert-success">
```

Comment:
Hi Luis, this is a new guideline, I'm sorry for not teaching you this sooner.

On lines 30, 35, 40, use `flash.discard(:notice)`, `flash.discard(:error)`, and `flash.discard(:alert)` respectively. This ensures that a flash notice is only displayed once. Without these, you may see the same flash message appear in a sequence of requests.